content/en/docs/reference/kubectl/kubectl.md

```diff
@@ -6,12 +6,11 @@ weight: 30
 
 ## {{% heading "synopsis" %}}
 
-
 kubectl controls the Kubernetes cluster manager.
 
- Find more information at: https://kubernetes.io/docs/reference/kubectl/overview/
+Find more information at: https://kubernetes.io/docs/reference/kubectl/overview/
```

Comment:
```suggestion
Find more information in [Command line tool](https://kubernetes.io/docs/reference/kubectl/)
(`kubectl`).
```

Reply:
I have only found this similar line in k/k repo, but it appears to be correct, so I update the current doc manually as you suggested.

https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/kubectl/pkg/cmd/cmd.go#L319